yarn.lock

```diff
@@ -3272,7 +3272,7 @@ glob@^10.0.0:
     package-json-from-dist "^1.0.0"
     path-scurry "^1.11.1"
 
-glob@^11.0.0, glob@^11.0.1:
+glob@^11.0.0:
```

Comment:
v11 comes in as a transitive dep of rimraf, which is just a dev dep here, so we don't need to worry about this.